spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/ribbon/apache/RibbonLoadBalancingHttpClient.java

```diff
@@ -64,9 +62,28 @@ public RibbonLoadBalancingHttpClient(final ILoadBalancer lb) {
 	}
 
 	@Override
+	public void initWithNiwsConfig(IClientConfig clientConfig) {
```

Comment:
You call this explicitly in tests. My guess is that this doesn't get called when not in a test, can you confirm?